core/src/apps/management/reset_device/__init__.py

```diff
@@ -111,31 +113,23 @@ async def reset_device(msg: ResetDevice) -> Success:
 
 
 async def _backup_slip39_basic(encrypted_master_secret: bytes) -> None:
+    group_threshold = 1
+
     # get number of shares
     await layout.slip39_show_checklist(0, BAK_T_SLIP39_BASIC)
-    shares_count = await layout.slip39_prompt_number_of_shares()
+    share_count = await layout.slip39_prompt_number_of_shares()
 
     # get threshold
     await layout.slip39_show_checklist(1, BAK_T_SLIP39_BASIC)
-    threshold = await layout.slip39_prompt_threshold(shares_count)
-
-    identifier = storage_device.get_slip39_identifier()
-    iteration_exponent = storage_device.get_slip39_iteration_exponent()
-    if identifier is None or iteration_exponent is None:
-        raise ValueError
+    share_threshold = await layout.slip39_prompt_threshold(share_count)
 
-    # generate the mnemonics
-    mnemonics = slip39.split_ems(
-        1,  # Single Group threshold
-        [(threshold, shares_count)],  # Single Group threshold/count
-        identifier,
-        iteration_exponent,
-        encrypted_master_secret,
-    )[0]
+    mnemonics = _get_slip39_mnemonics(
+        encrypted_master_secret, group_threshold, [(share_threshold, share_count)]
```

Comment:
very tiny nit:
```suggestion
        encrypted_master_secret, group_threshold, ((share_threshold, share_count),)
```
we prefer tuples to lists if we at all can
in some places it's because tuples are immutable so semantically more appropriate (e.g., when checking membership in a list of constants)
in other places, like here, because a tuple is just one allocation but a list is two.